src/models/Plan.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const schema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  createdAt: {
+    type: Date,
```

Comment:
Set this to `required true` and your post method function will be responsible to ensure it gets the current timestamp and include in the `create` method